mkosi

```diff
@@ -747,14 +753,14 @@ def disable_kernel_install(args, workspace):
     for f in ("50-dracut.install", "51-dracut-rescue.install", "90-loaderentry.install"):
         os.symlink("/dev/null", os.path.join(workspace, "root", "etc/kernel/install.d", f))
 
-def invoke_dnf(args, workspace, repositories, base_packages, boot_packages):
+def invoke_dnf_or_yum(args, workspace, repositories, base_packages, boot_packages, command="dnf"):
 
     repos = ["--enablerepo=" + repo for repo in repositories]
 
     root = os.path.join(workspace, "root")
-    cmdline = ["dnf",
+    cmdline = [command,
                "-y",
-               "--config=" + os.path.join(workspace, "dnf.conf"),
+               "--config=" + os.path.join(workspace, command + ".conf"),
                "--best",
                "--allowerasing",
```

Comment:
`--best` and `--allowerasing` do not exist for Yum. Not setting the weakdeps policy for DNF makes the images much fatter, too.